torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -183,9 +181,8 @@ inline Tensor log_softmax(const Tensor& input, int64_t dim,
 }
 } // namespace detail
 
-inline Tensor log_softmax(const Tensor& input, LogSoftmaxFuncOptions options,
-                          c10::optional<torch::Dtype> dtype = c10::nullopt) {
-  return detail::log_softmax(input, options.dim(), dtype);
+inline Tensor log_softmax(const Tensor& input, LogSoftmaxFuncOptions options) {
```

Comment:
functional.py has default values for all options in:
`def log_softmax(input, dim=None, _stacklevel=3, dtype=None)`
so maybe
```suggestion
inline Tensor log_softmax(const Tensor& input, LogSoftmaxFuncOptions options = {}) {
```

Reply:
replied in an earlier comment